app/src/main/res/layout/fragment_report.xml

```diff
@@ -1,25 +1,139 @@
+                    android:layout_marginTop="12dp"
+                    app:endIcon="@drawable/ic_location"
+                    app:header="@string/location"
+                    app:layout_constraintTop_toBottomOf="@id/input_locality"
+                    app:required_input="true" />
+
+                <com.humara.nagar.ui.custom_views.FormInputWithHeader
+                    android:id="@+id/input_comment"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="12dp"
+                    android:layout_marginBottom="10dp"
+                    app:header="@string/comment"
+                    app:layout_constraintTop_toBottomOf="@id/input_location"
+                    app:minimumHeight="250" />
+
+                <androidx.constraintlayout.widget.ConstraintLayout
+                    android:id="@+id/addImageLayout"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="20dp"
+                    app:layout_constraintStart_toStartOf="@+id/input_comment"
+                    app:layout_constraintTop_toBottomOf="@+id/input_comment">
+
+                    <ImageView
+                        android:id="@+id/add_btn"
+                        android:layout_width="25dp"
+                        android:layout_height="25dp"
+                        app:srcCompat="@drawable/ic_add_button"
+                        app:layout_constraintBottom_toBottomOf="parent"
+                        app:layout_constraintStart_toStartOf="parent"
+                        app:layout_constraintTop_toTopOf="parent" />
+
+                    <TextView
+                        android:id="@+id/textView"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginStart="15dp"
+                        android:fontFamily="@font/open_sans_bold"
+                        android:text="@string/addImage"
+                        android:textColor="#4F4F4F"
+                        android:textSize="16sp"
+                        app:layout_constraintEnd_toEndOf="parent"
+                        app:layout_constraintHorizontal_bias="0.0"
+                        app:layout_constraintStart_toEndOf="@+id/add_btn"
+                        app:layout_constraintTop_toTopOf="parent" />
+
+                    <TextView
+                        android:id="@+id/textView_subtitle"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginTop="5dp"
+                        android:fontFamily="@font/open_sans_semibold"
+                        android:text="@string/uploadImageDescription"
+                        android:textColor="#828282"
```

Comment:
hardcoded color code. Color and string values should always be extracted in respective res files

Reply:
Fixed